src/styles/blocks/clock.scss

```
@@ -0,0 +1,36 @@
+.stopwatch {
+  position: relative;
+  height: $clock-size;
+  width: $clock-size;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1vmin dotted black;
+  border-radius: 50%;
+
+  &__minute-hand {
+    position: absolute;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: $minute-hand-color;
+    transform-origin: 50% 100%;
+    animation: rotate 600s linear infinite;
+  }
+
+  &__second-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: $second-hand-color;
+    animation: rotate 10s linear infinite;
+    transform-origin: 50% 100%;
+  }
+
+  &__axis {
+    border-radius: 50%;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: $axis-color;
```

you can use a circle `mixin` to avoid code duplicate

```
@mixin circle($width, $color) {
  width: $width;
  height: $width;
  background: $color;
  border-radius: $width/2;
}
``` 

```suggestion
  &__axis {
    @include circle(5vmin, $axis-color);
```